docs/upgrade/upgrade-opensource/upgrade-guide-from-5.2-to-5.4/upgrade-guide-from-5.2-to-5.4-generic.rst

```diff
@@ -63,26 +63,23 @@ We recommend upgrading the Monitoring Stack to the latest version.
 
 .. note::
    
-   In ScyllaDB 5.4, Raft-based consistent cluster management for existing
+   In ScyllaDB 5.4, Raft-based consistent cluster management for existing 
    deployments is enabled by default.
+   If you want the consistent cluster management feature to be disabled in 
+   ScyllaDB 5.4, you must update the configuration **before** you upgrade 
+   from 5.2 to 5.4:
 
-   If you want to disable the consistent cluster management feature in ScyllaDB 5.4,
-   you must update the configuration **before** you start the upgrade procedure from 
-   5.2 to 5.4:
+   #. Set ``consistent_cluster_management: false`` in the ``scylla.yaml`` 
+      configuration file on each node in the cluster.
+   #. Start the upgrade procedure.
 
-   #. Ensure that the schema is synchronized in the cluster by executing 
-      :doc:`nodetool describecluster </operating-scylla/nodetool-commands/describecluster>` 
-      on each node and ensuring that the schema version is the same on all nodes.
-   #. Perform a :doc:`rolling restart </operating-scylla/procedures/config-change/rolling-restart/>`, 
-      updating the ``scylla.yaml`` file for **each node** in the cluster before 
-      restarting it to  disable the ``consistent_cluster_management`` option:
+   Consistent cluster management **cannot** be disabled in version 5.4 if it was 
+   enabled in version 5.2 in one of the following ways:
 
-       .. code-block:: yaml
-
-          consistent_cluster_management: false
-
-      When all the nodes in the cluster are updated and restarted, you can start
-      the upgrade procedure.
+   * Your cluster was created in version 5.2. In 5.2, consistent cluster 
+     management is enabled in new clusters.
```

Comment:
The reason why a new cluster would enable Raft in 5.2 is because the default `conf/scylla.yaml` that we provide in this version contains `consistent_cluster_management: true`.

Which means that the user could create a 5.2 cluster without Raft, by modifying `conf/scylla.yaml` (or providing a custom one in the first place) and removing `consistent_cluster_management: true` (or set it to `false`).

Maybe the "in one of the following ways:" part is not necessary? We could just say --- if you enabled in 5.2, you cannot disable it during upgrade. You can only prevent it from getting enabled if you didn't yet have it enabled.

Reply:
Makes sense